src/frontend/src/pages/HomePage/WorkPackagesByTimelineStatus.tsx

```diff
@@ -61,24 +94,49 @@ const WorkPackagesByTimelineStatus: React.FC = () => {
             <Card
               variant="outlined"
               key={wbsPipe(wp.wbsNum)}
-              sx={{ minWidth: 'fit-content', mr: 3, background: theme.palette.background.default }}
+              sx={{ width: '450px', mr: 3, background: theme.palette.background.default, borderRadius: '16px' }}
             >
-              <CardContent sx={{ padding: 3 }}>
-                <Link
-                  variant="h6"
-                  component={RouterLink}
-                  to={`${routes.PROJECTS}/${wbsPipe(wp.wbsNum)}`}
-                  sx={{ marginBottom: 2 }}
-                >
-                  {wbsPipe(wp.wbsNum)} - {wp.name}
-                </Link>
-                <DetailDisplay label="End Date" content={datePipe(wp.endDate)} paddingRight={2} />
-                <DetailDisplay label="Progress" content={percentPipe(wp.progress)} paddingRight={2} />
-                <DetailDisplay label="Engineering Lead" content={fullNamePipe(wp.projectLead)} paddingRight={2} />
-                <DetailDisplay label="Project Manager" content={fullNamePipe(wp.projectManager)} paddingRight={2} />
-                <Typography>
-                  {wp.expectedActivities.length} Expected Activities, {wp.deliverables.length} Deliverables
-                </Typography>
+              <CardContent sx={{ padding: 2 }}>
+                <Grid container spacing={2}>
+                  <Grid container item xs={9} spacing={1} zeroMinWidth>
+                    <Grid item xs={12} zeroMinWidth>
+                      <Typography justifyContent={'flex-end'} sx={{ textDecoration: 'underline', align: 'right' }}>
+                        <Link component={RouterLink} to={`${routes.PROJECTS}/${wbsPipe(wp.wbsNum)}`}>
+                          <Typography variant="h6">
+                            {projectWbsPipe(wp.wbsNum)} - {wp.projectName}{' '}
+                          </Typography>
+                        </Link>
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={12} zeroMinWidth>
+                      <Link component={RouterLink} to={`${routes.PROJECTS}/${wbsPipe(wp.wbsNum)}`} noWrap={true}>
+                        <Typography variant="h5">
+                          {wbsPipe(wp.wbsNum)} - {wp.name}
+                        </Typography>
+                      </Link>
+                    </Grid>
+                    <Grid item xs={12} zeroMinWidth>
+                      <Typography fontWeight={'normal'}>
+                        {datePipe(wp.startDate) + ' |-----' + wp.duration + 'wks-----| ' + datePipe(wp.endDate)}
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={6} zeroMinWidth>
+                      <Chip sx={{ marginTop: 1 }} icon={<Construction />} label={fullNamePipe(wp.projectLead)} />
```

Comment:
extracting these chips below the two other containers will probably add the margin top that you want.